src/shogun/base/ShogunEnv.h

```diff
@@ -106,6 +106,17 @@ namespace shogun
 			return sg_graph_backend;
 		}
 
+		/** set the global graph backend
+		 *
+		 * @return the backend enum
+		 */
+		void set_graph_backend(GRAPH_BACKEND backend)
+		{
+			// if (!shogun::graph::available_backends().count(backend))
+				sg_graph_backend = backend;
+			// else
+			// 	error("Backend not available.");
+		}
```

Comment:
@vigsterkr the problem with setting is that we need to add something like `shogun::graph::available_backends()` to ShogunEnv